make-docs.bat

```diff
@@ -0,0 +1,41 @@
+@echo off
+rem Batch script to clean docs artifacts and generate Sphinx HTML documentation
+
+rem Remove docs/notebooks/_data/
+if exist docs\notebooks\_data\ (
+    rd /s /q docs\notebooks\_data\
+)
+
+rem Remove docs/notebooks/.ipynb_checkpoints/
+if exist docs\notebooks\.ipynb_checkpoints\ (
+    rd /s /q docs\notebooks\.ipynb_checkpoints\
+)
+
+rem Remove specific rst files from docs/apidoc
+del /q docs\apidoc\xhydro*.rst
+del /q docs\apidoc\modules.rst
+
+rem Remove French locale .mo files
+del /q docs\locales\fr\LC_MESSAGES\*.mo
+
+rem Generate API documentation
+sphinx-apidoc -o docs\apidoc --private --module-first src\xhydro
+
+rem Generate gettext files
+sphinx-build -b gettext docs docs\_build\gettext
```

Comment:
We discussed the idea of having a separate `.bat` to generate the translation, similar to how `make docs` and `make initialize-translations` is separated for Linux. Would that be possible?

I don't expect Windows users to want to generate the full documentation locally too often, just update the .po files.

Reply:
Makes sense. I'll split these when I have a minute.